### PR TITLE
block production time

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
service to return block production time

### CODE REVIEW

File: src/modules/health/health.controller.ts [hunk 1]
```diff
@@ -38,4 +44,23 @@ export class HealthController {
...
+  @ApiNotFoundResponse({description: 'Block production is unhealthy.'})
+  public async blockProduction(@Res() res: Response): Promise<any> {
+    const diff = await this.healthService.blockProduction();
+    if (diff) {
+      res.status(HttpStatus.NO_CONTENT).send();
+    }
+    res.status(HttpStatus.NOT_FOUND).send();
+  }
+
+  @Get('node-dropped')
```
Comment (evgeny-s) [inline src/modules/health/health.controller.ts:59]: This is the scope of the other ticket, right?

File: src/modules/health/health.service.ts [hunk 2]
```diff
@@ -51,9 +51,43 @@ export class HealthService {
...
     const finalized = Number(await this.api.derive.chain.bestNumberFinalized());
     const best = Number(await this.api.derive.chain.bestNumber());
     return {finalized, best};
   }
+
+  private blockTime(extrinsic: any) {
+    return new Promise((resolve, reject) => {
+      extrinsic.forEach((element) => {
+        if (element.method.section === 'timestamp') {
+          // eslint-disable-next-line prefer-destructuring
```
Comment (evgeny-s) [inline src/modules/health/health.service.ts:87]: Can we use destructuring instead of disabling of linter?

**Review by evgeny-s — CHANGES_REQUESTED** [review]

File: src/modules/health/health.controller.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.controller.ts:59]: Yes.

File: src/modules/health/health.service.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply src/modules/health/health.service.ts]: Removed.

File: src/modules/health/health.controller.ts (hunk 1 above)
Comment (evgeny-s) [reply src/modules/health/health.controller.ts:59]: ok, let's exclude these changes from this PR

**Review by evgeny-s — APPROVED** [review]

**Pull request merged**